jenkins/new-scan-engine/Jenkinsfile-sysdig-cli-scanner

```diff
@@ -1,18 +1,43 @@
 pipeline {
-    agent any
 
-    stages {
-        stage('Scan image') {
-            steps {
-                withCredentials([usernamePassword(credentialsId: 'sysdig-secure-api-credentials', passwordVariable: 'SECURE_API_TOKEN', usernameVariable: '')]) {
+    parameters { 
+        string(name: 'DOCKER_REPOSITORY', defaultValue: 'sysdigcicd/cronagent', description: 'Name of the image to be built (e.g.: sysdiglabs/dummy-vuln-app)') 
+        string(name: 'GIT_REPOSITORY', defaultValue: 'https://github.com/sysdiglabs/secure-inline-scan-examples.git', description: 'Name of the repository with the Dockerfile to be built (e.g.: https://github.com/sysdiglabs/secure-inline-scan-examples.git)') 
+    }
+
+  agent any
+  stages {
+    stage('Cloning Git') {
+      steps {
+        git branch: 'main', url: "${params.GIT_REPOSITORY}"
+      }
+    }
+    stage('Build Image') {
+      steps {
+          sh "docker build -f ./jenkins/new-scan-engine/Dockerfile -t ${DOCKER_REPOSITORY} ./jenkins/new-scan-engine/"
+        }
+    }
+    stage('Scan image') {
+      steps {
+        withCredentials([usernamePassword(credentialsId: 'sysdig-secure-api-token', passwordVariable: 'SECURE_API_TOKEN', usernameVariable: '')]) {
+          sh '''
+            VERSION=$(curl -L -s https://download.sysdig.com/scanning/sysdig-cli-scanner/latest_version.txt)
+            curl -LO "https://download.sysdig.com/scanning/bin/sysdig-cli-scanner/${VERSION}/linux/amd64/sysdig-cli-scanner"
+            chmod +x ./sysdig-cli-scanner
+            ./sysdig-cli-scanner --apiurl https://eu1.app.sysdig.com docker://${DOCKER_REPOSITORY}
```

Comment:
may we add any comment suggesting to customize the endpoint/region?
https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges

just suggesting

Reply:
Great suggestion! PTAL